.github/workflows/concrete_python_tests_linux.yml

```diff
@@ -111,5 +111,6 @@ jobs:
             export COMPILER_BUILD_DIRECTORY=/build
             export KEY_CACHE_DIRECTORY=/tmp/KeySetCache
             make pytest
```

Comment:
clean in the Makefile before and after ?

Reply:
Not in the makefile, but done.